web/app/js/components/MetricsTable.jsx

```diff
@@ -17,6 +16,8 @@ import { withContext } from './util/AppContext.jsx';
 
 const columnDefinitions = (resource, showNamespaceColumn, PrefixedLink) => {
   let isAuthorityTable = resource === "authority";
+  let isMultiResourceTable = resource === "multi_resource";
+  let getResourceDisplayName =  isMultiResourceTable ? displayName : d => d.name;
 
   let nsColumn = [
```

Comment:
whoops good catch; this title should be "Resource" not "Multi Resource". Forgot to change this over :)
and yeah services are deliberately in there because I couldn't decide whether to include them or not. I can hide them for now, we can revisit!